**Math 345M Homework 2**

**Author: Cole Pendergraft**

**Chapter 3 Problems: 86, 87, 89, 112, 116 b-f**

**Chapter 4 Problems: 71, 72, 74, 78, 80**

In [23]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

**Problem 3.86:** Roll two fair dice separately. Each die has six faces.

**a) List the sample space**

The sample space is a list of all possible outcomes of an event, and since both die have a sample space of S = {1, 2, 3, 4, 5}, then the sample space for two fair die rolled separately would be S = { {1, 1}, {1, 2}, {1, 3}, {1, 4}, {1, 5}, {1, 6}, {2, 1}, {2, 2}, {2, 3}, {2, 4}, {2, 5}, {2, 6}, {3, 1}, {3, 2}, {3, 3}, {3, 4}, {3, 5}, {3, 6}, {4, 1}, {4, 2}, {4, 3}, {4, 4}, {4, 5}, {4, 6}, {5, 1}, {5, 2}, {5, 3}, {5, 4}, {5, 5}, {5, 6}, {6, 1}, {6, 2}, {6, 3}, {6, 4}, {6, 5}, {6, 6} }

**b) Let A be the event that either a three or a four is rolled first, followed by an even number. Find P(A).**

As per the book, we calculate P(A) by taking the number of favorable outcomes for event A and divide it by the total number of possible outcomes. Event A dictates that either a three or four is rolled, and then an even number, so our sample space is S(A) = { {3, 2}, {3, 4}, {3, 6}, {4, 2}, {4, 4}, {4, 6} }, and |S(A)| = 6. Our total number of possible outcomes is 36, so we have that P(A) = 6/36 = 1/6.

**c) Let B be the event that the sum of the two rolls is at most seven. Find P(B).**

Event B dictates that the sum of our two rolls is less than or equal to 7. Thus, our sample space is S(B) = { {1, 1}, {1, 2}, {1, 3}, {1, 4}, {1, 5}, {1, 6}, {2, 1}, {2, 2}, {2, 3}, {2, 4}, {2, 5}, {3, 1}, {3, 2}, {3, 3}, {3, 4}, {4, 1}, {4, 2}, {4, 3}, {5, 1}, {5, 2}, {6, 1} }, and |S(B)| = 21. Once again, our total number of outcomes is 36, so we have P(B) = 21/36 = 7/12.

**d) In words, explain what "P(A|B)" represents. Find P(A|B).**

This is the conditional probability of A given B. In our terms, it is the probability that that either a 3 or 4 is rolled first and followed by an even number given that the sum of the two rolls is at most seven. The book details that we can solve this using the following equation:

P(A|B) = P(A AND B)/P(B)

P(A AND B) is relatively straightforward to find, all we must do is remove entries from the sample space S(A) in part (b) that violate the condition in B, and this will leave us with just the outcomes where a 3 or 4 is rolled first and followed by an even number AND the sum of the two rolls is less than or equal to 7. This leaves us with S(A AND B) = { {3, 2}, {3, 4}, {4, 2} }, |S(A AND B)| = 3, and P(A AND B) = 3/36. Thus, we have P(A|B) = (3/36) / (21/36).

**e) Are A and B mutually exusive events? Explain your answer in one to three complete sentences, including numerical justification.**

No they are not mutually exclusive. To be mutually exclusive, it must be impossible for the two events to occur at the same time, but there is no barrier that prevents the outcomes from both summing to a value less than or equal to seven, and starting with a 3 or 4 and then followed by an even number. Numerically, if these events were mutually exclusive we would have P(A AND B) = 0, as it would be impossible for both events to occur simultaneously. We have demonstrated that P(A AND B) = 3/36, which is not 0, so these two events cannot be mutually exclusive.

**f) Are A and B independent events? Explain your answser in one to three complete sentences, including numerical justification.**

In words, we can consider two events as independent if the knowledge that one occurred does not affect the chance that the other occurs. We can see this criteria as being violated due to the fact the two die values needing to sum to seven affects how many possible outcomes can happen. If these were independent, adding the criteria from B to the criteria from A should yield no change in the number of outcomes for either.

Numerically speaking, as per the book, we know that two events are independent if the following are true:

- P(A|B) = P(A)

- P(B|A) = P(B)

- P(A AND B) = P(A)P(B)

The first criteria for independence has not been met, as we have P(A|B) = 1/7, and P(A) = 1/6, so these two events cannot be independent.

**Problem 3.87:** A special deck of cards has ten cards. Four are green, three are blue, and three are red. When a card is picked, its color of it is recorded. An experiment consists of first picking a card and then tossing a coin.

**a) List the sample space.**

Due to the fact that the deck has 10 cards, it may seem that we would have a lot of possible outcomes here, but we actually do not. Despite there being ten cards in the deck, the only thing we care about is the color of the card drawn, of which there are three possibilities. The other factor is whether the coin is heads or tails, so we only actually have 6 possible outcomes. These are S = { {R, H}, {B, H}, {G, H}, {R, T}, {B, T}, {G, T} }

**b) Let A be the event that a blue card is picked first, followed by landing a head on the coin toss. Find P(A).**

Ther is only one outcome in which we get a blue card and then a heads, so we have that P(A) = 1/6.

**c) Let B be the event that a red or green is picked, followed by landing a head on the coin toss. Are the events A and B mutually exclusive? Explain your answer in one to three sentences, includeing numerical justification.**

Yes, these events are mutually exclusive. In words, we cannot both have a red or green AND a blue card, if the card is blue, then it isn't red or green, and if it is red or green, then it isn't blue.

Numerically, mutually exclusive events should have P(A AND B) = 0. In this case, A AND B translates to the event that a blue card is picked first, followed by a head, AND that a red or green is picked. Thus, we just need to select the elements from A that fit the criteria in B, of which there are none. Thus, P(A AND B) = 0 and these events are mutually exclusive.

**d) Let C be the event that a red or blue is picked, followed by landing a head on the coin toss. Are the events A and C mutually exclusive? Explain your answer in one to three complete sentences, including numerical justification.**

No, these events are not mutually exclusive. Event C is the same as event A with just one additional criteria, so all outcomes in event A still fall under the umbrella of event C.

Numerically, mutually exclusive events should have P(A AND B) = 0. To test this, we can select the elements of A that fulfill the requirements of C, of which there is one. The {B, H} element in A is a blue card and then a head, which fits within the criteria for C and thus P(A AND B) is nonzero, so events C and A are not mutually exclusive. 

**Problem 3.89:** An experiment consists of tossing a nickel, a dime, and a quarter. Of interest is the side the coin lands on.

**a) List the sample space.**

S = { {H, H, H}, {H, H, T}, {H, T, T}, {H, T, H} {T, T, T}, {T, T, H}, {T, H, H}, {T, H, T} }

**b) Let A be the event that there are at least two tails. Find P(A).**

Lets select the outcomes from our sample space that fit within the criteria for event A. We then have S(A) = { {H, T, T}, {T, T, T}, {T, T, H}, {T, H, T} } and |S(A)| = 4. Thus, we have P(A) = 4/8 = 1/2.

**c) Let B be the event that the first and second tosses land on heads. Are the events A and B mutually exclusive? Explain your answer in one to three complete sentences, including justification.**

Yes, they are mutually exclusive. Event A dictates that we have AT LEAST two tails, but event B dictates that we have exactly 2 heads. If we are flipping three coins, it is wholly impossible for us to at once have both 2 heads and at least 2 tails, thus these events are mutually exclusive. Numerically, this amounts to P(A AND B) = 0.

**Problem 3.112:** Table 3.22 identifies a group of children by one of four hair colors, and by type of hair.

| Hair Type | Brown | Blond | Black | Red | Totals |
|-----------|-------|-------|-------|-----|--------|
| Wavy | 20 | 5 | 15 | 3 | 43 |
| Straight | 80 | 15 | 65 | 12 | 172 |
| Totals | 100 | 20 | 80 | 15 | 215 |

**a) Complete the table.** 

See table above

**b) What is the probability that a randomly selected child will have wavy hair?**

We are seeking the probability a child selected randomly from amongst the 215 will have wavy hair regardless of hair color. There are a total of 43 children with wavy hair, so our probability is then 43/215.

**c) What is the probablility that a randomly selected child will have either brown or blond hair?**

We are seeking the probability that a child selected randomly from amongst the 215 will have either brown OR blond hair regardless of hair type. There are a total of 100 brown haired children and 20 blond haired children, so we have (100 + 20) / 215 = 120/215.

**d) What is the probability that a randomly selected child will have wavy brown hair?**

We are seeking the probability that a child selected randomly from amongst the 215 will have the wavy hair type AND brown hair color. There are only 20 children that fit within this criteria, so our probability is 20/215.

**e) What is the probability that a randomly selected child will have red hair, given that he or she has straight hair?**

This one is tricky, it seems like it may be an AND statement, but is actually a conditional P(A|B). We know that we can compute P(A|B) with the equation

P(A|B) = P(A AND B) / P(B).

So our first step is to compute P(A AND B). Event A dictates that the child will have red hair, and event B dictates that the child will have straight hair. Thus, we have 12 children that fit into A AND B, and P(A AND B) = 12/215.

Since B is the event that the child will have straight hair, which amounts to 172 kids. Thus, P(B) = 172/215.

Now, we can calculate P(A|B) = (12/215)/(172/215) = 12/172

**f) If B is the event of a child having brown hair, find the probability of the complement of B.**

We have 100 kids with brown hair, so P(B) = 100/215. We know that the complement of B is such that P(B) + P(B') = 1, so we can say that P(B') = 1 - P(B) = 1 - 100/215 = 115/215

**g) In words, what does the complement of B represent?**

In words, the complement of B, labeled B', is the event that contains all possible outcomes that are NOT in B. 

**Problem 3.116:** A box of cookies contains three chocolate and seven butter cookies. Miguel randomly selects a cookie and eats it. Then he randomly selects another cookie and eats it. (How many cookies did he take?)

**b) Are the probabilities for the flavor of the SECOND cookie that Miguel selects independent of his first selection? Explain.**

No, they are not. We are making selections without replacement here, as Miguel eats the cookie after choosing it. By doing this, he changes the total number of cookies remaining in the box, and by extension the probabilities of choosing each type of cookie, so the two events cannot be independent.

For example, we start off with P(Chocolate) = 3/10 = 30% chance and P(Butter) = 7/10 = 70% chance. Let's suppose that Miguel selects a Butter cookie first and eats it. We are now left with 9 cookies in the box and the following probabilities: P(Chocolate) = 3/9 = 33% chance and P(Butter) = 67% chance. So by selecting a cookie from the box, Miguel alters the probabilities for the second cookie by making the selection of a chocolate more likely, which means these two events cannot be independent.

**C) For each complete path through the tree, write the event it represents and find the probabilities.**

We aren't using a tree, but we can still accomplish this task. We are making two choices, and the possible outcomes are S = { {C, C}, {C, B}, {B, C}, {B, B} }, so we have 4 possible outcomes. 

Suppose our outcome is {C, C}:

We have 3/10 chance of choosing the first chocolate cookie, and then a 2/9 chance of choosing the second. Thus, P({C, C}) = 3/10 * 2/9 = 6/90

If  our outcome is {C, B}:

We have a 3/10 chance of choosing the chocolate cookie first, and then a 7/9 chance to choose a butter cookie second. Thus, P({C, B}) = 3/10 * 7/9 = 21/90

If our outcome is {B, B}:

We have a 7/10 chance of choosing the first butter cookie, and then a 6/9 chance of choosing the second butter cookie. Thus, P({B, B}) = 7/10 * 6/9 = 42/90

And if our outcome is {B, C}:

We have a 7/10 chance of choosing the butter cookie first, and then a 3/9 chance of choosing the chocolate cookie second. Thus, we have P({B,C}) = 7/10 * 3/9 = 21/90

**d) Let S be the event that both cookies selected were the same flavor. Find P(S).**

P(S) is essentially equivalent to P(CC OR BB), as those are the only two outcomes in which both cookies match, and we are looking for one of those two outcomes. Thus, P(S) = P(CC OR BB) = P(CC) + P(BB) = 6/90 + 42/90 = 48/90

**e) Let T be the event that the cookies selected were different flavors. Find P(T) by two different methods: by using the complement rule and by using the branches of the tree. Your answers should be the same with both methods.**

Since we aren't using a tree for this problem, I will only answer the question using the complement rule. Event T is essentially the complement of event S from the previous problem, as every event in T is the same as every event NOT in S. Since we should have that P(T) + P(S) = 1, we know that P(T) = 1 - P(S) = 1 - 48/90 = 42/90.

**f) Let U be the event that the second cookie selected is a butter cookie. Find P(U).**

Event U in this case is essentially equivalent to BB OR CB, as those are the only two potential outcomes in which a butter cookie is chosen second. Thus, we have P(U) = P(BB OR CB) = P(BB) + P(CB) = 42/90 + 21/90 = 63/90

**Problem 4.71:** A game involves selecting a card from a regular 52-card deck and tossing a coin. The coin is a fair coin and is equally likely to land on tails or heads.

- If the card is a face card, and the coin lands on heads, you win $6.

- If the card is a face card, and the coin lands on tails, you win $2.

- If the card is not a face card, you lose $2 regardless of the coin.

**a) Find the expected value for this game (expected net gain or loss).**

For this calculation we want to start by computing the probabilities of each outcome:

That you draw 1 of the 12 face cards in a 52 card deck and your coin lands on heads = 12/52 * 1/2 = 12/104 = 3/26

That you draw 1 of the 12 face cards in a 52 card deck and your coin lands on tails = 12/52 * 1/2 = 12/104 = 3/26

That you draw 1 of the 40 non-face cards in a 52 card deck = 40/52 = 20/26

Now attaching the monetary values to each probability to calculate expected profit/loss for each outcome:

We have a 3/26 chance of winning 6 dollars = 18/26 dollars expected profit

We have a 3/26 chance of winning 2 dollars = 6/26 dollars expected profit

We have a 20/26 chance of losing 2 dollars = 40/26 dollars expected loss

Now we simply combine our three individual expecte values into one overall expected value as follows:

18/26 + 6/26 - 40/26 = -16/26 dollars expected loss

**b) Explain what your calculations indicate about your long-term average profits and losses on this game.**

This game is likely to lose you a lot more money than it will win you money. Each run of the game is likely to lose you about 16/26 dollars, so in the long run it isn't a viable option if one wants to make money. Even when you do win the potential gain of 18/26 dollars at best doesn't even come close to the potential losses of 40/26 dollars.

**c) Should you play this game to win money?**

Definitely not, as you play more rounds of this game you will trend towards losses over profits.

**Problem 4.72:** You buy a lottery ticket that costs 10 dollars per ticket. There are only 100 tickets available to be sold in this lottery. In this  lottery there is one 500 dollar prize, two 100 dollar prizes, and four 25 dollar prizes. Find your expected gain or loss.

Lets begin by calculating the probabilities of the particular prizes:

We have a 1/100 chance of getting the 500 dollar prize

We have a 2/100 chance of getting one of the 100 dollar prizes

We have a 4/100 chance of getting one of the 25 dollar prizes

In all other outcomes, you will lose 10 dollars.

Now we can attach monetary values to each outcome:

500 * 1/100 = 5 dollars

100 * 2/100 = 2 dollars

25 * 4/100 = 1 dollar

Potential loss = 10 dollars. Since we are only purchasing a single lottery ticket, we don't have to do 10 * 93/100, as no matter what the worst that could happen is that we lose 10 dollars for the ticket.

Now all that is left is to combine everything and calculate out expected loss/gain:

5 + 2 + 1 - 10 = -2

So you can expect to lose about 2 dollars per game.

**Problem 4.74:** Suppose that you are offered the following "deal". You roll a die. If you roll a six, you win 10 dollars. If you roll a four or five, you win 5 dollars. If you roll a 1, 2, or 3, you pay 6 dollars.

**a) What are you ultimately interested in here (the value of the roll or the money you win)?**

Winning money is entirely dependent on the value of the die, so ultimately we care about the value of the roll.

**b) In words, define the Random Variable X.**

The random variable X will be the value of the roll, and we will be calculating probabilities of certain values.

**c) List the values that X may take on.**

Since X is the value of the roll, it can be 1, 2, 3, 4, 5, or 6.

**d) Construct a PDF.**

Since each value 1 to 6 is represented only 1 time on an average die, there is a 1/6 chance for each face value. Thus, our PDF is:

| X | P(X) |
|---|------|
| 1 | 1/6 |
| 2 | 1/6 |
| 3 | 1/6 |
| 4 | 1/6 |
| 5 | 1/6 |
| 6 | 1/6 |

**e) Over the long run of playing this game, what are your expected average winnings per game?**

To calculate this, we first need to compute our individual winnings and losings based on probability:

We have a 1/6 chance to roll a 1, which would net us a 6 dollar loss, so we have 6 * 1/6 = 1 dollar expected loss

The above is true for a roll of 2, so we have 6 * 1/6 = 1 dollar expected loss

The above is true for a roll of 3, so we have 6 * 1/6 = 1 dollar expected loss

If we roll a 4, we stand to win 5 dollars, so we have 5 * 1/6 = 5/6 dollars expected gain

The above is true for a roll of 5, so we have 5 * 1/6 = 5/6 dollars expected gain

If we roll a 6, we stand to win 10 dollars, so we have 10 * 1/6 = 10/6 dollars expected gain

Now we must simply combine the expected values based on their sign:

5/6 + 5/6 + 10/6 - 3 = 20/6 - 18/6 = 2/6 = 0.33 dollar average gain.

**f) Based on numerical values, should you take the deal? Explain your decision in complete sentences.**

Yep! In part e above, we demonstrated that you are more likely to win money each game ( about 33 cents ), so it is in the players interest to take the game.

**Problem 4.78:** People visiting video rental stores often rent more than one DVD at a time. The probability distribution for DVD rentals per customer at Video To Go is given in the following table. There is a five-video limit per customer at this store, so nobody ever rents more than five DVDs.

| x | P(x) |
|---|------|
| 0 | 0.03 |
| 1 | 0.50 |
| 2 | 0.24 |
| 3 | 0.66 |
| 4 | 0.07 |
| 5 | 0.04 |

**a) Describe the random variable X in words.**

The random variable x is simply the number of movies rented by a customer at this video rental store.

**b) Find the probability that a customer rents three DVDs.**

The total of all P(x) elements should be 1, and we currently have 0.03 + 0.5 + 0.24 + 0.07 + 0.04 = 0.34, so the probability of chosing 3 movies is 0.66.

**c) Find the probability that a customer rents at least four DVDs.**

We are seeking P(x >= 4) which will be the combined probabilities of all values greater than or equal to 4, which would be 0.07 + 0.04 = 0.11. So the probability that the customer rents at least four DVDs is 0.11.

**d) Find the probability that a customer rents at most two DVDs.**

We are seeking P( x <= 2) which will be the combined probabilities of all values less than or equal to 2, which would be 0.03 + 0.50 + 0.24 = 0.77.

Another shop, Entertainment Headquarters, rents DVDs and video games. The probability distribution for DVD rentals per customer at thsi shop is given as follows. They also have a five-DVD limit per customer.

| x | P(x) |
|---|------|
| 0 | 0.35 |
| 1 | 0.25 |
| 2 | 0.20 |
| 3 | 0.10 |
| 4 | 0.05 |
| 5 | 0.05 |

**e) At which store is the expected number of DVDs rented per customer higher?**

We are essentially computing an average here of each different store, and then comparing those averages. We compute this average in the same way we computed average loss/gain in some of the previous problems.

$ \mu = \Sigma x*P(x) $

Starting with Video To Go:

$ \mu = 0 + 0.5 + 2(0.24) + 3(0.66) + 4(0.07) + 5(0.04) $

And now with Entertainment HQ:

$ \mu = 0 + 0.25 + 2(0.20) + 3(0.10) + 4(0.05) + 5(0.05) $

In [24]:
print("VTG avg:",0+0.5+(2*0.24)+(3*0.66)+(4*0.07)+(5*0.04))
print("EHQ avg:",0+0.25+(2*0.20)+(3*0.10)+(4*0.05)+(5*0.05))

VTG avg: 3.4400000000000004
EHQ avg: 1.4000000000000001


So it would seem that, on average, there are more rentals at Video To Go than there are at Entertainment HQ.

**f) If Video to Go estimates that they will have 300 customers next week, how many DVDs do they expect to rent next week? Answer in sentence form.**

Well, if Video To Go is expecting 300 customers, and if Video To Go customers rent about 3 movies on average, then it is reasonable to assume that each customer will rent approximately 3 movies which would equate to about 900 video rentals.

**g) If Video to Go expects 300 customers next week, and Entertainment HQ projects they will have 420 customers, for which stoer is the expected number of DVD rentals for next week higher? Explain.**

So, we can preform this calculation just like the previous part. Our average is a pretty solid measure of how many films we can expect each customer to rent, so multiplying that average by the number of expected customers gives us a pretty good idea of what we can expect when it comes to sales. We know already that VTG is expected to rent something like 900 DVDs to 300 customers. For EHQ, we are expecting about 420 customers, but those customers are only expected to rent about 1 movie per individual, so EHQ only has an expected rental count of 420, so we would expect VTG to have the higher weekly rentals.

**h) Which of the two video stores experiences more variation in the number of DVD rentals per customer? How do you know that?**

Well, this question is essentially asking us which store has a greater standard deviation. We can interpret "which store experiences more variance in the number of DVD rentals per customer" as "which store has a greater customer to customer deviation". Standard deviation is a value we can calulate using the following equation:

$\sigma = \sqrt{\Sigma(x - \mu)^2p(x)}$

Instead of attempting to guess at which has a higher deviation by looking at the data, I am just going to preform the calculation so I can determine for sure which has the highest deviation customer to customer.

Thus, for VTG we have

$\sigma = \sqrt{\Sigma(x - \mu)^2p(x)}$

| x | P(x) | $(X - \mu)^2p(x)$ |
|---|------|-------------------|
| 0 | 0.03 | $(0 - 3.44)^2(0.03)$ |
| 1 | 0.50 | $(1 - 3.44)^2(0.50)$ |
| 2 | 0.24 | $(2 - 3.44)^2(0.24)$ |
| 3 | 0.66 | $(3 - 3.44)^2(0.66)$ |
| 4 | 0.07 | $(4 - 3.44)^2(0.07)$ |
| 5 | 0.04 | $(5 - 3.44)^2(0.04)$ |

In [25]:
a = ((0-3.44)**2)*(0.03)
b = ((1 - 3.44)**2)*(0.50)
c = ((2 - 3.44)**2)*(0.24)
d = ((3 - 3.44)**2)*(0.66)
e = ((4 - 3.44)**2)*(0.07)
f = ((5 - 3.44)**2)*(0.04)

print("VTG standard deviation:", (a + b + c + d + e + f)**(1/2))

VTG standard deviation: 2.0190453189564614


Now, for EHQ we have:

| x | P(x) | $(X - \mu)^2p(x)$ |
|---|------|-------------------|
| 0 | 0.35 | $(0 - 1.4)^2(0.35)$ |
| 1 | 0.25 | $(1 - 1.4)^2(0.25)$ |
| 2 | 0.20 | $(2 - 1.4)^2(0.20)$ |
| 3 | 0.10 | $(3 - 1.4)^2(0.10)$ |
| 4 | 0.05 | $(4 - 1.4)^2(0.05)$ |
| 5 | 0.05 | $(5 - 1.4)^2(0.05)$ |

In [26]:
a = ((0-1.4)**2)*(0.35)
b = ((1 - 1.4)**2)*(0.25)
c = ((2 - 1.4)**2)*(0.20)
d = ((3 - 1.4)**2)*(0.10)
e = ((4 - 1.4)**2)*(0.05)
f = ((5 - 1.4)**2)*(0.05)

print("EHQ standard deviation:", (a + b + c + d + e + f)**(1/2))

EHQ standard deviation: 1.42828568570857


So VTG experiences more deviation with rentals, and I know this because I did the calculation above.

**Problem 4.80:** Florida State University has 14 statistics classes scheduled for its Summer 2013 term. One class has space available for 30 students, eight class have space for 60 students, one class has space for 70 students, and four classes have space for 100 students.

**a) What is the average class size assuming each class is filled to capacity?**

We can perform this calculation in the same way we performed the gain/loss calculations in previous problems by substituing the class size in as the random variable X instead of the money.

We have 1/14 classes with seating size of 30, so we have 1/14 * 30 = 30/14

We have 8/14 classes with a seating size of 60, so we have 8/14 * 60 = 480/14

We have 1/14 classes with a seating size of 70, so we have 1/14 * 70 = 70/14

We have 4/14 classes with a seating size of 100, so we have 4/14 * 100 = 400/14

And thus, we have

(30 + 480 + 70 + 400)/14 = 980/14 = 70

So the average class size is 70.

**b) Space is available for 980 students. Suppose that each class is filled to capacity and select a statistics student at random. Let the random variable X equal the size of the student's class. Define the PDF for X.**

| X | P(X) |
|---|------|
| 30 | 1/14 |
| 60 | 8/14 |
| 70 | 1/14 |
|100| 4/14 |

**c) Find the mean of X.**

We have actually already preformed this calculation in part a, since X is our class size, finding the mean of X is the same thing as finding the average class size as we did in part a. We started by multiplying each random variable X by the probability of that class size, and then summing all those values. Thus, we have

(30 + 480 + 70 + 400)/14 = 980/14 = 70

**d) Find the standard deviation of X.**

The calculation for the standard deviation of X is

$\sigma = \sqrt{\Sigma(x - \mu)^2p(x)}$

So, updating our PDF, we have

| X | P(X) | $(X - \mu)^2p(x)$|
|---|------|------------------|
| 30 | 1/14 | $(30 - 70)^2 * (1/14)$ |
| 60 | 8/14 | $(60 - 70)^2 * (8/14)$ |
| 70 | 1/14 | $(70 - 70)^2 * (1/14)$ |
|100| 4/14 | $(100 - 70)^2 * (4/14)$ |

and we actually get to use a coding cell here to do some math!

In [2]:
print((30 - 70)**2 * (1/14))
print((60 - 70)**2 * (8/14))
print((70 - 70)**2 * (1/14))
print((100 - 70)**2 * (4/14))

114.28571428571428
57.14285714285714
0.0
257.1428571428571


So now all we have left to is combine these values and take the square root of the result:

$\sqrt{114.29 + 57.14 + 0 + 257.17}$

In [4]:
(114.29 + 57.14 + 257.17)**(1/2)

20.70265683432926

So the standard deviation of class sizes is

$\sigma = 20.703$